study.md

```diff
@@ -50,7 +50,25 @@ A kitchen has many items.
     rack, and cinnamon, nutmeg, and vanilla on another shelf.
 
 ```json
-<!-- your answer here -->
+<!--
+{
+    "kitchen": {
+      "cupboard": {
+        "bowls": "10",
+        "plates": "10"
+      },
+      "stove": {
+        "heat1": "small",
+        "heat2": "medium",
+        "heat3": "large",
+        "heat4": "extra large"
+      },
+      "spice rack": {
+        "shelf1": "chives, lemon grass, mint",
```

Comment:
You would probably want to represent these items as an array, what you have here is one long string